uwblueprint/supportive-housing
# Submit Login/Signup with Enter Key

## Notion task link


N/A



## Implementation description
* Added HTML form tag to `Login.tsx` and `Signup.tsx`



## Steps to test
1. Create a new user 
2. Verify that Login and Signup submit when the enter key is pressed



## What should reviewers focus on?
* Should their be some feedback given to the user when they submit this way? (Normally the button changes when clicked)


## Checklist
